.github/workflows/auth-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Auth Tests
+on:
+  workflow_dispatch:
+  schedule:
+    # Wednesday 1400
+    - cron:  '0 14 * * 3'
```

Comment:
Probably not worth scheduling yet, we can just run on demand until its ready to go.